python/python/bystro/search/utils/annotation.py

```diff
@@ -94,7 +94,7 @@ def from_path(
     "field": "\t",
     "allele": "/",
     "position": "|",
-    "overlap": "\\",
+    "overlap": chr(31),
```

Comment:
overlap delimiter is used in master annotator, and this matches the Seq::Output::Delimiters value (it is a control character; we should discuss better ways of handling overlapping values and nested relationships in the future).

Reply:
If this piece of configuration is shared between perl and python code, what would you think about pulling it out into a config file that both can draw from so they can't get out of sync?

Failing that, we should probably at least document that this is tied to a definition in the annotator.

Reply:
Will do in a followup PR. We can't afaik put non-rendered characters in yaml, but that may be the wrong control character to use. Will need to this more about this. I think this points out the need to move away from .tsv